deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/exception/GenericExceptionMapper.java

```diff
@@ -48,6 +49,21 @@ public Response toResponse(Throwable t) {
                 sb.append("\n").append(sw.toString());
                 return Response.serverError().entity(sb.toString()).build();
             }
+        } else if (t instanceof ConstraintViolationException) {
```

Comment:
I see a bit of duplicate code in toResponse method which i think could be extracted out and put in helper method. However, I see it's existing code, so leave it up to you.